README.md

```diff
@@ -12,6 +12,20 @@ https://docs.cossacklabs.com/pages/java-and-android-howto/
 2. Open `java-example` in IntelliJ IDEA.
 3. Run `main.java`.
 
+_Note_: if you want to use bundled (or copied) `lib/libthemis_jni.{so or dylib}`
+you will have to add `LD_LIBRARY_PATH=lib` (even better, use the absolute path to `lib`)
+to environment variables of the `Application` `main` configuration, so the JVM
+will be able to find it.
+
+If you cannot run the example app because of error like this
+```
+Exception in thread "main" java.lang.UnsatisfiedLinkError: <some path>/libthemis_jni.so: libthemis.so.0: cannot open shared object file: No such file or directory
+```
+you will have to add `/usr/local/lib` (or wherever the `libthemis` is stored)
+to the same `LD_LIBRARY_PATH` variable. You can combine two directories like this:
+`LD_LIBRARY_PATH=lib:/usr/local/lib` (just separate with `:`).
+This magic variable may be used on both Linux and macOS.
```

Comment:
yeah, this might be quite different on macOS. We have following note in docs: https://docs.cossacklabs.com/themis/languages/java/installation-desktop/#installing-stable-version-on-macos